app/models/admin/tuition_fee.rb

```diff
@@ -0,0 +1,6 @@
+class Admin::TuitionFee < ApplicationRecord
+
+  belongs_to :group, :class_name => Admin::AdmissionFeesStatus
```

Comment:
Why do use `Admin::AdmissionFeesStatus` as class name ? Is this group  related to `Admin::AdmissionFeesStatus` model or `Admin::AdmissionTuitionGroup` model ?